src/App.js

```diff
@@ -1,5 +1,52 @@
+const { Console } = require('@woowacourse/mission-utils');
+const { MESSAGE, CHOICE } = require('./Constant');
+const BaseballGameTools = require('./BaseballGameTools');
+
 class App {
-  play() {}
+  constructor() {
+    this.answerArray = [];
+  }
+
+  shuffleNumber() {
+    this.answerArray = BaseballGameTools.getThreeNumber();
+  }
+
+  enterGuessMode() {
+    Console.readLine(MESSAGE.ASK_GUESS, (playerGuess) => {
+      BaseballGameTools.errorIfInvalidGuessFormat(playerGuess);
+      const baseballResult = BaseballGameTools
+        .getBaseballResultMessage(this.answerArray, playerGuess);
+      Console.print(baseballResult);
+
+      if (baseballResult === MESSAGE.THREE_STRIKE) {
+        Console.print(MESSAGE.GAME_CLEAR);
+        this.enterReplayChooseMode();
+      } else {
+        this.enterGuessMode();
+      }
+    });
+  }
+
+  enterReplayChooseMode() {
+    Console.readLine(MESSAGE.ASK_RESTART, (playerChoice) => {
+      BaseballGameTools.errorIfInvalidChoiceFormat(playerChoice);
+
+      if (playerChoice === CHOICE.PLAY_AGAIN) {
+        this.shuffleNumber();
+        this.enterGuessMode();
```

Comment:
이 두 라인이 반복적으로같이 사용되니까 아래처럼 루틴이나 커맨드라는 키워드로 뺴내서 하나의 메소드로 묶어주는게 좋을 것 같아요!
```js
playroutine(){
        this.shuffleNumber();
        this.enterGuessMode();
}
```
```suggestion
        this.playroutine()
```

Reply:
그렇네요, 앞으로는 여러 개의 함수가 항상 묶여서 사용되는 상황이면 여러 개의 함수를 합친 하나의 함수를 따로 선언하도록 하겠습니다. 감사합니다!

Reply:
화이팅이에요:)